info.rkt

```diff
@@ -4,7 +4,7 @@
                "shlex"
                "base"))
 (define build-deps '("scribble-lib" "racket-doc"))
-(define scribblings '(("scribblings/drracket-cmdline-args.scrbl" ())))
+(define scribblings '(("scribblings/drracket-cmdline-args.scrbl" () ("DrRacket Plugin"))))
```

Comment:
Judging from how other category names are (mostly) plural, I think this should be "DrRacket Plugins". 

A perhaps better approach is to introduce a new symbol that `raco setup` understands. This would be better discussed on Discourse.

Reply:
agreed. changed to DrRacket Plugins

Reply:
I agree that a new standard symbol may one a good approach. I will review how this is implemented in raco